x-pack/metricbeat/magefile.go

```diff
@@ -0,0 +1,297 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build mage
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/thehivecorporation/log"
```

Comment:
Tried to run it locally and this package was not found. Is this package used on purpose?

Reply:
I have just pushed normal `log` library on the dependencies in my last commit. I think that it always autoimports "this" as it's the one I use the most

Reply:
This is already fixed